tests/st/codegen/torch/test_torch_codegen_cross_core.py

```
@@ -611,6 +656,112 @@ def test_split_aiv_fused_mixed_codegen_vs_golden(
+    v: pl.Tensor,
+    out: pl.Out[pl.Tensor],
+):
+    """High-level (@pl.jit) analog of SplitAivQkPvProgram: out = ((q @ k^T) * 2) @ v.
+
+    Both matmuls live OUTSIDE the ``pl.split_aiv`` region (cube work); the region
+    shards the ``raw = q @ k^T`` Tensor to each AIV lane, scales the half, and
+    gathers it back to a full Tensor consumed by the second (cube) matmul.
+    """
+    with pl.at(level=pl.Level.CORE_GROUP, name_hint="tqkpv"):
+        raw = pl.matmul(q, k, b_trans=True, out_dtype=pl.FP32)  # Tensor [M, N] = q @ k^T
+        for aiv_id in pl.split_aiv(2, mode=pl.SplitMode.UP_DOWN):
```

Ruff (0.15.20)

[warning] Loop control variable `aiv_id` not used within loop body

Rename unused `aiv_id` to `_aiv_id`

(B007)